docs/dev/build_windows.md

```diff
@@ -6,20 +6,20 @@ Supported configurations:
 - Windows 10 x86 64-bit or higher with Visual Studio 2019 or higher build for X64 architecture.
 - Windows on ARM (shortly WoA) to build for ARM64 architecture. OpenVINO was validated on [Windows DevKit 2023](https://developer.qualcomm.com/hardware/windows-on-snapdragon/windows-dev-kit-2023)
 
-## Software requirements 
+## Software requirements
 
 - [CMake](https://cmake.org/download/) 3.13 or higher
 - Microsoft Visual Studio 2019 or higher, version 16.3 or later
-  > **NOTE**: Native Microsoft Visual Studio for WoA is available since 2022. 
+  > **NOTE**: Native Microsoft Visual Studio for WoA is available since 2022.
```

Comment:
"has been available" if we want to indicate the timeframe - present perfect. Same in the line below :)
Also, "since version 3.11" would look better for the python one.